Jellyfin.Plugin.Anime/Providers/AniList/AniListApi.cs

```diff
@@ -16,7 +16,7 @@ namespace Jellyfin.Plugin.Anime.Providers.AniList
     /// Based on the new API from AniList
     /// 🛈 This code works with the API Interface (v2) from AniList
     /// 🛈 https://anilist.gitbooks.io/anilist-apiv2-docs
-    /// 🛈 THIS IS AN UNOFFICAL API INTERFACE FOR EMBY
+    /// 🛈 THIS IS AN UNOFFICAL API INTERFACE FOR JELLYFIN
```

Comment:
Summary of changes in this file:
* Fixed indentation (mostly in graphql)
* Moved some functions to be methods of the Media object
* Update comments